viz.yaml

```diff
@@ -265,11 +271,14 @@ publish:
     template: fullpage
     publisher: page
     depends:
+      sourcesansprofont: sourcesansprofont
```

Comment:
I'm mucking around with this to try to figure out why the footer looks dumb again....I don't think this is it...I'll remove